library/core/src/primitive_docs.rs

```diff
@@ -587,8 +587,10 @@ mod prim_pointer {}
 /// There are two syntactic forms for creating an array:
 ///
 /// * A list with each element, i.e., `[x, y, z]`.
-/// * A repeat expression `[x; N]`, which produces an array with `N` copies of `x`.
-///   The type of `x` must be [`Copy`].
+/// * A repeat expression `[expr; N]` where `N` is how many times to repeat `expr` in the array. `expr` must either be:
+///
+///   * A value of type `Copy`
```

Comment:
`Copy` is not a type.
```
A value of a type implementing the [`Copy`] trait
```
might be more accurate

Reply:
Applied :+1: